### PR TITLE
Added information on our tech stack to the readme

### PR DESCRIPTION
**Pull request by mcoffey-mt**

### CODE REVIEW

File: README_template.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+This is a skeleton project from which to create new kotlin projects from.
```
Comment (chubberlisk) [inline README_template.md:1]: I think we don't need to keep this as we can refer to it in [the template repository](https://github.com/ministryofjustice/hmpps-template-kotlin#readme).

File: README.md [hunk 2]
```diff
@@ -5,47 +5,32 @@
 [![Docker Repository on Quay](https://quay.io/repository/hmpps/hmpps-template-kotlin/status "Docker Repository on Quay")](https://quay.io/repository/hmpps/hmpps-template-kotlin)
 [![API docs](https://img.shields.io/badge/API_docs_-view-85EA2D.svg?logo=swagger)](https://hmpps-template-kotlin-dev.hmpps.service.justice.gov.uk/webjars/swagger-ui/index.html?configUrl=/v3/api-docs)
 
-This is a skeleton project from which to create new kotlin projects from.
+## Infrastructure Tech Stack
+Information on the technology stack chosen for the HMPPS-Integration-API project
 
-# Instructions
+### [Cloud Platform](https://user-guide.cloud-platform.service.justice.gov.uk/#cloud-platform-user-guide)
+A hosting platform which offers numerous tools such as logging, monitoring and alerting for our services.
```
Comment (chubberlisk) [inline README.md:12]: Might be worth adding that this is an MoJ thing that's built on top of AWS.

File: README.md [hunk 3]
```diff
@@ -5,47 +5,32 @@
...
-<https://github.com/ministryofjustice/cloud-platform-environments/tree/main/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-template-kotlin>
+### [Circle CI](https://circleci.com/developer)
+_This may yet be switched for Github Actions depending on investigation_
 
-Copy this folder, update all the existing namespace references, and submit a PR to the CloudPlatform team. Further instructions from the CloudPlatform team can be found here: <https://user-guide.cloud-platform.service.justice.gov.uk/#cloud-platform-user-guide>
+### [AWS](https://aws.amazon.com/)
+Services utilise AWS features through cloud platform.
 
-## Renaming from HMPPS Template Kotlin - github Actions
+Used for our build platform, responsible for executing workflows to build, validate, test and deploy our project.
```
Comment (chubberlisk) [inline README.md:29]: I don't think this is quite true, our pipeline is responsible for building, validating, testing and deploying - it runs the commands to do those things and it's more like it tells things like AWS that we want to push a new image version.

**Review by chubberlisk — APPROVED** [review]
Nice work - left a couple of comments but feel free to merge afterwards!

File: README.md (hunk 2 above)
Comment (mcoffey-mt) [reply README.md:12]: I agree 100%!

File: README.md (hunk 3 above)
Comment (mcoffey-mt) [reply README.md:29]: Sorry, this should have been caught by my self-review. It's in the wrong place, should be within CircleCI. Good spot!

**Pull request merged**